dvc/remote/gdrive.py

```diff
@@ -396,14 +385,14 @@ def _gdrive_download_file(
         param = {"id": item_id}
         # it does not create a file on the remote
         gdrive_file = self._drive.CreateFile(param)
-        bar_format = (
-            "Downloading {desc:{ncols_desc}.{ncols_desc}}... "
-            + Tqdm.format_sizeof(int(gdrive_file["fileSize"]), "B", 1024)
-        )
+
         with Tqdm(
-            bar_format=bar_format, desc=progress_desc, disable=no_progress_bar
-        ):
-            gdrive_file.GetContentFile(to_file)
+            total=int(gdrive_file["fileSize"]),
```

Comment:
does it make a meta call? can we avoid that? API calls are very expensive.

Reply:
good spot. This actually isn't needed because the callback will eventually set/update the `total`.